docs/tutorial.md

```diff
@@ -112,7 +112,7 @@ mothulity_dbaser ~/databases_directory --silva-119
 ## Setting Persistent Database Path
 
 
-```mothulity``` needs to know where the databases live.You can specify the path each time you run the analysis with:
+```mothulity``` needs to know where the databases live.You can specify the path each time you run the analysis using arguments:
```

Comment:
You changed the description in a good way. It is easier to understand this is about an argument. Nevertheless, I would go with the full command ```mothulity --align-database ...``` just like you did with rest of the examples. I mean in a way that suitable for copy-paste.

Reply:
My idea was that here we just comment on **argument** itself, not the 'full command'. The problem is that if we want to provide the full command it would be:
mothulity <path_to_data> -r <mode> -n <name> **--align-database ~/databases_directory/silva.nr_v119.align** <- this is a minimal example. The argument --align-database cannot be executed without those additional parameters.
We can also stick to simple:
mothulity <<other_arguments>>  **--align-database ~/databases_directory/silva.nr_v119.align**
Not sure what solution would be the most intuitive one.

Reply:
Let us assume that users love copy-paste. Now with your changes I can see this clearly :) So even though I 100% agree this is logical, let's make it ready for copy-paste, full command